chord_metadata_service/restapi/api_views.py

```diff
@@ -52,87 +47,31 @@ async def service_info(_request):
-            },
-            "diseases": {
-                # count is a number of unique disease terms (not all diseases in the database)
-                "count": diseases_count,
-                "term": diseases_stats
-            },
-            "individuals": {
-                "count": individuals_count,
-                "sex": {k: individuals_sex.get(k, 0) for k in (s[0] for s in pheno_models.Individual.SEX)},
-                "karyotypic_sex": {
-                    k: individuals_k_sex.get(k, 0) for k in (s[0] for s in pheno_models.Individual.KARYOTYPIC_SEX)
-                },
-                "taxonomy": stats_for_field(patients_models.Individual, "taxonomy__label"),
-                "age": individuals_age,
-            },
-            "phenotypic_features": {
-                # count is a number of unique phenotypic feature types (not all pfs in the database)
-                "count": phenotypic_features_count,
-                "type": stats_for_field(pheno_models.PhenotypicFeature, "pftype__label")
-            },
-            "experiments": {
-                "count": experiments_count,
-                "study_type": stats_for_field(experiments_models.Experiment, "study_type"),
-                "experiment_type": stats_for_field(experiments_models.Experiment, "experiment_type"),
-                "molecule": stats_for_field(experiments_models.Experiment, "molecule"),
-                "library_strategy": stats_for_field(experiments_models.Experiment, "library_strategy"),
-                "library_source": stats_for_field(experiments_models.Experiment, "library_source"),
-                "library_selection": stats_for_field(experiments_models.Experiment, "library_selection"),
-                "library_layout": stats_for_field(experiments_models.Experiment, "library_layout"),
-                "extraction_protocol": stats_for_field(experiments_models.Experiment, "extraction_protocol"),
-            },
-            "experiment_results": {
-                "count": experiment_results_count,
-                "file_format": stats_for_field(experiments_models.ExperimentResult, "file_format"),
-                "data_output_type": stats_for_field(experiments_models.ExperimentResult, "data_output_type"),
-                "usage": stats_for_field(experiments_models.ExperimentResult, "usage")
-            },
-            "instruments": {
-                "count": instruments_count,
-                "platform": stats_for_field(experiments_models.Experiment, "instrument__platform"),
-                "model": stats_for_field(experiments_models.Experiment, "instrument__model")
-            },
-        }
-    }
+    phenopackets_summary, experiments_summary = await asyncio.gather(
+        dt_phenopacket_summary(phenopackets, low_counts_censored=False),
+        dt_experiment_summary(experiments, low_counts_censored=False),
+    )
 
-    return Response(r)
+    return Response({
```

Comment:
New response shape is not handled properly in the bento_web overview, all charts are empty.

Reply:
did you try the PR mentioned in the description? it is a breaking change so this is expected without it